nupic/research/fast_temporal_memory.py

```diff
@@ -182,9 +182,9 @@ def cellsForColumn(self, column):
     """
     self._validateColumn(column)
 
-    start = self.cellsPerColumn * column
-    end = start + self.cellsPerColumn
-    return set([ConnectionsCell(idx) for idx in range(start, end)])
+    start = int(self.cellsPerColumn * column)
+    end = int(start + self.cellsPerColumn)
```

Comment:
@chetan51 
I don't think this change will work if you're using the fast_temporal_memory with the network api. Someone probably knows why, but when I was testing this, column was always just a `numpy.int64`, not one of the C++ connections, so the `self.getCellIndex(column)` would throw an error since column wouldn't have the `.idx` attribute.